test-suite/src/test/scala/org/scalajs/testsuite/javalib/ConcurrentHashMapTest.scala

```diff
@@ -0,0 +1,531 @@
+package org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.scalajs.jasminetest.JasmineTest
+
+import java.util.concurrent.ConcurrentHashMap
+
+object ConcurrentHashMapTest extends JasmineTest {
+
+  describe("java.util.concurrent.ConcurrentHashMap") {
+
+    def expectNullPointerException(f: => Unit) = {
+      expect({
+          try {
+            f
+            false
+          } catch {
+            case err: NullPointerException => true
+            case _: Throwable => false
+          }
+      }).toBeTruthy
+    } 
+
+    it("should store strings") {
+      val chm = new ConcurrentHashMap[String, String]()
+
+      expect(chm.size()).toEqual(0)
+      chm.put("ONE", "one")
+      expect(chm.size()).toEqual(1)
+      expect(chm.get("ONE")).toEqual("one")
+      chm.put("TWO", "two")
+      expect(chm.size()).toEqual(2)
+      expect(chm.get("TWO")).toEqual("two")
+    }
+
+    it("should store integers") {
+      val chm = new ConcurrentHashMap[Int, Int]()
+
+      chm.put(1, 1)
+      expect(chm.size()).toEqual(1)
+      val one = chm.get(1)
+      expect(one).toEqual(1)
```

Comment:
All these `1`'s ^^ Maybe it would be clearer/safer to use values that are different than keys.